staging/src/k8s.io/apiserver/pkg/endpoints/handlers/response.go

```diff
@@ -95,12 +102,14 @@ func transformResponseObject(ctx context.Context, scope RequestScope, req *http.
 			}
 
 			// renegotiate under the internal version
+			trace.Step("Renegotiating internal version")
```

Comment:
Same with this one.

Reply:
Happy to drop this one.

Reply:
I think this is now inline with the previous comments - so please drop this and the one mentioned below.

Reply:
Done